ParkingDatabase/src/main/java/com/example/parkingdatabase/manager/ParkingService.java

```diff
@@ -0,0 +1,47 @@
+package com.example.parkingdatabase.manager;
+
+import com.example.parkingdatabase.Color;
+import com.example.parkingdatabase.dao.entity.Car;
+import com.example.parkingdatabase.dao.entity.CarRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ParkingService {
```

Comment:
This service should be injected to the Controller and you should pass data from requests to add it to database (I suggest to replace it with simple HashMap for now)